tests/integration/test.bulk_get.js

```diff
@@ -74,6 +74,98 @@ adapters.forEach(function (adapter) {
       });
     });
 
+    it('test bulk get with bad revision specified', function (done) {
+      var db = new PouchDB(dbs.name);
+      db.put({_id: 'foo', val: 1}).then(function () {
+        db.bulkGet({
+          docs: [
+            {id: 'foo', rev: 'BAD_REV'}
+          ]
+        }).then(function (resp) {
+          var result = resp.results[0];
+          result.id.should.equal('foo');
+          if (adapter === 'local') {
```

Comment:
Ideally we should have the same error API between local and remote, but if not then you can make this assertion a little less specific to avoid the switch here, can just check that it is an error doc